hazelcast/src/test/java/com/hazelcast/internal/management/ClearWanQueuesTest.java

```diff
@@ -0,0 +1,62 @@
+package com.hazelcast.internal.management;
+
+import com.eclipsesource.json.JsonObject;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.instance.Node;
+import com.hazelcast.internal.management.operation.ClearWanQueuesOperation;
+import com.hazelcast.internal.management.request.ChangeWanStateRequest;
+import com.hazelcast.internal.management.request.ClearWanQueuesRequest;
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelTest.class})
+public class ClearWanQueuesTest extends HazelcastTestSupport {
+
+    private ManagementCenterService managementCenterService;
+
+    @Before
+    public void setUp() {
+        HazelcastInstance hz = createHazelcastInstance();
+        Node node = getNode(hz);
+        managementCenterService = node.getManagementCenterService();
+    }
+
+    @Test
+    public void testOperationDefaultConstructor() {
+        ClearWanQueuesOperation operation = new ClearWanQueuesOperation();
+        assertNotNull(operation);
+    }
+
```

Comment:
I don't see a test where the actual behavior is being tested. So you you send a ClearWanQueuesRequest, but you don't verify that the wan queue actually has been cleared.